test/table.js

```diff
@@ -3,7 +3,7 @@ const { JSDOM } = jsdom;
 require("iconv-lite").encodingExists("foo");
 const tableSortJs = require("../public/table-sort");
 
-function createTestTable(testTableData, classTags="") {
+function createTestTable(testTableData, classTags="", invisibleIndex=[]) {
```

Comment:
I should change the function to take an object instead of variables; however this is out of the scope of this PR.